wa/framework/configuration/plugin_cache.py

```diff
@@ -73,10 +73,7 @@ def add_configs(self, plugin_name, values, source):
         if self.is_global_alias(plugin_name):
             self.add_global_alias(plugin_name, values, source)
             return
-        for name, value in values.iteritems():
-            self.add_config(plugin_name, name, value, source)
 
-    def add_config(self, plugin_name, name, value, source):
```

Comment:
BTW this means there's no longer an 'add_config' method - I couldn't see any callers so I judged it was fine..